build/configs/artik053/scripts/flash.ld

```diff
@@ -72,6 +72,7 @@ SECTIONS
 
 	.text : {
 		_stext = ABSOLUTE(.);
+		_ram_kernel_text_start = ABSOLUTE(.);
```

Comment:
1. I don't think this is good way. Because it requires unnecessary and wrong code. (This is not ram kernel text region)
How about adding new config?
When we enable the new config, then we check _ram_kernel_text_start.

2. How about renaming this like `_stext`? `_stext_ram`? I want to use the same policy for them.